draft-kuehlewind-shmoo-remote-fee.md

```diff
@@ -64,14 +64,14 @@ these trends the IETF started requiring registration as "participant" (in contra
 an "observer") for remote participation, still without any registration fee applied.
 
 With the recent move to fully online meetings, however, there is no longer a distinction
-between remote and on-site participants. Since costs still have to be covered, there is
+between remote and on-site participants. Since IETF meeting costs and other costs still have to be covered, there is
 the need for a meeting fee for remote participants, which risks the removal of the free
 remote option.
 
-This change led to concerns about the impact both on those who
-regularly remotely attend meetings, as well as people looking to
-attend IETF meetings for the first time.  In both cases, even a small
-registration fee can be a barrier to participation.
+This possibility raised concerns about the potential impact both on those who
+regularly remotely attend IETF meetings, as well as people considering 
+attending an IETF meeting for the first time, of the effects of IETF meeting fees.  In both cases, even a small
+registration fee could potentially be a barrier to participation for some people.
```

Comment:
I like the previous slightly stronger wording given we know of cases that didn't join because of the fee (and did not about the waiver or did feel they should be using it).

Reply:
+1